.github/workflows/recipe.yaml

```diff
@@ -46,25 +44,24 @@ jobs:
           docker run --rm --name flex -d -v $PWD/plugin/recipes:/var/www/flex/var/repo/private/monsieurbiz/sylius-cms-page-plugin -p 80:80 docker.pkg.github.com/monsieurbiz/docker/symfony-flex-server:latest contrib official
           docker ps
 
+      - run: mkdir -p /home/runner/{.composer/cache,.config/composer}
+
```

Comment:
@maximehuran Isn't it blocking the cache if this `mkdir` is done before next step?

Isn't it better to keep it after?